transport/whisper/envelopes.go

```diff
@@ -27,65 +25,70 @@ type EnvelopesMonitorConfig struct {
 	EnvelopeEventsHandler          EnvelopeEventsHandler
 	MaxAttempts                    int
 	MailserverConfirmationsEnabled bool
-	IsMailserver                   func(enode.ID) bool
+	IsMailserver                   func(whispertypes.EnodeID) bool
 	Logger                         *zap.Logger
 }
 
 // EnvelopeEventsHandler used for two different event types.
 type EnvelopeEventsHandler interface {
 	EnvelopeSent([][]byte)
 	EnvelopeExpired([][]byte, error)
-	MailServerRequestCompleted(common.Hash, common.Hash, []byte, error)
-	MailServerRequestExpired(common.Hash)
+	MailServerRequestCompleted(whispertypes.Hash, whispertypes.Hash, []byte, error)
```

Comment:
`Hash` is not a Whisper-only type. It's used thoroughly. We can replace it with `[]byte` probably if we need to?

Reply:
I just had to create a folder in `/types` for `HexBytes` (to replace `hexutils.Bytes`), I guess I could now move this there?